### DISCUSSION TITLE
zopen upgrade xxx question

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#233] [General]
This morning I issued  `zopen upgrade git ` and it worked reporting that it was installing git, bash, etc.  

It would be nice if the upgrade notice were to provide insight into the actual upgrade.  I also suspect the installs were because they are pre/co-req's of git and may or may not have needed a new install for an upgrade.

Questions I have no answers to.

**Comment by lbdyck** [comment]
It would also be nice to have a `zopen log` to view recent zopen actions.

**Reply by IgorTodorovskiIBM** [reply]
I like this idea, a log of recent zopen actions would help us maintain audit trails and identify potential issues. This thought also came up in our recent meeting with sys admins.

**Reply by IgorTodorovskiIBM** [reply]
@DevonianTeuchter is adding logging support into zopen (as well as a lot of other great additions) and will be demo'ing it in the next guild call.

**Reply by lbdyck** [reply]
SWEET

**Comment by lbdyck** [comment]
sample zopen upgrade git:

![image](https://user-images.githubusercontent.com/42328411/226320237-89c0b6e3-f060-46bb-939d-fd32c7e349b2.png)

**Reply by IgorTodorovskiIBM** [reply]
@DevonianTeuchter - does your fork include information about the upgrade when upgrading?
